homeassistant/components/aurora_abb_powerone/const.py

```diff
@@ -1,5 +1,7 @@
 """Constants for the Aurora ABB PowerOne integration."""
 
+from .config_flow import CONF_TYPE_SERIAL
```

Comment:
I think this might be causing the circular import.

Normal procedure is to create the shared constants in `const.py` then import those elsewhere.  I don't think we should import consts from other files back into `const.py`